app/src/main/res/layout/add_habit_dialog_box.xml

```diff
@@ -43,33 +44,34 @@
         android:id="@+id/confirm_habit"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="30dp"
+        android:layout_marginTop="16dp"
         android:layout_marginBottom="16dp"
         android:background="@color/habit_list_text"
         android:text="@string/confirm"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="@+id/habit_date_selection"
-        app:layout_constraintTop_toBottomOf="@+id/habit_date_selection" />
+        app:layout_constraintTop_toBottomOf="@+id/thursday_button" />
```

Comment:
There are lots of these little changes, which are just formatting changes for each screen to adjust for the new buttons.